examples/echobot2.py

```diff
@@ -29,22 +29,27 @@
 # Define a few command handlers. These usually take the two arguments bot and
 # update. Error handlers also receive the raised TelegramError object in error.
 def start(bot, update):
+    """Send a message when the command /start is issued."""
     update.message.reply_text('Hi!')
 
 
 def help(bot, update):
+    """Send a message when the command /help is issued."""
     update.message.reply_text('Help!')
 
 
 def echo(bot, update):
+    """Echo the user message."""
     update.message.reply_text(update.message.text)
 
 
 def error(bot, update, error):
-    logger.warn('Update "%s" caused error "%s"' % (update, error))
+    logger.warn('Update "%s" caused error "%s"', update, error)
```

Comment:
Can you change this to `logger.warning`? `Logger.warn` is deprecated.
https://docs.python.org/3/library/logging.html#logging.Logger.warning